packages/flutter_tools/test/fuchsia/fuchsa_device_test.dart

```diff
@@ -58,6 +60,56 @@ void main() {
     });
   });
 
+  group('Fuchsia device artifact overrides', () {
+    final MockFile devFinder = MockFile();
```

Comment:
nit: do the setup in `setup` methods. When using IDE features like "run a single test" or the command line version, the test framework still needs to invoke the group closure. If the work is done in setup, it can be skipped if none of the child tests are run.

In this case its not too bad, but its a good general practice that we're not doing a great job of following currently

Reply:
Done.